test/functional/job_invocations_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_plugin_helper'
+
+class JobInvocationsControllerTest < ActionController::TestCase
+  test 'too much failed hosts' do
+    hosts = FactoryGirl.create_list(:host, 300, :comment => "benchmark-#{Foreman.uuid}")
+
+    template = FactoryGirl.create(:job_template)
+    task = FactoryGirl.create(:some_task, :result => 'error')
+    template_invocations = FactoryGirl.build_list(
+      :template_invocation,
+      300,
+      :template => template,
+      # :job_invocation => invocation,
```

Comment:
Should this be commented out? If so, it could be probably deleted altogether, right?